README.adoc

```diff
@@ -16,48 +16,53 @@
 :common-includes: https://raw.githubusercontent.com/OpenLiberty/guides-common/master
 = Creating an application with Bean Validation
 
-You'll explore how to use Bean Validation in Open Liberty to create an application to validate data.
+You'll explore how to use Bean Validation in Open Liberty to create an application to 
+validate data.
 
-// =================================================================================================
+// ==================================================================================
 // What you'll learn
-// =================================================================================================
+// ==================================================================================
 == What you'll learn
 
-You will learn the basics of writing and testing an application that uses Bean Validation and the
-new functionality of Bean Validation 2.0. The application uses Bean Validation to validate data
-from a form or servlet request and to ensure that the request meets the defined constraints.
+You will learn the basics of writing and testing an application that uses Bean 
+Validation and the new functionality of Bean Validation 2.0. The application 
+uses Bean Validation to validate data from a form or servlet request and to 
+ensure that the request meets the defined constraints.
 
 === What is Bean Validation?
 
-Bean Validation is a Java specification that simplifies data validation and error checking. Bean
-Validation uses a standard way to validate data stored in JavaBeans. Validation can be performed
-manually or with integration with other specifications and frameworks, such as CDI, JPA, or JSF.
-To set rules on data, apply constraints by using annotations or XML configuration files. Bean
-Validation provides both built-in constraints and the ability to create custom constraints. Bean
-Validation allows for validation of both JavaBean fields and methods. For method-level validation,
-both the input parameters and return value can be validated.
-
-=== What's new in Bean Validation 2.0
-
-Several built-in constraints are included in Bean Validation 2.0, such as `@Email`, `@NotBlank`,
-`@Positive`, and `@Negative`. You can also specify constraints on type parameters.
+Bean Validation is a Java specification that simplifies data validation and error 
+checking. Bean Validation uses a standard way to validate data stored in 
+JavaBeans. Validation can be performed manually or with integration with other 
+specifications and frameworks, such as CDI, JPA, or JSF. To set rules on data, 
+apply constraints by using annotations or XML configuration files. Bean
+Validation provides both built-in constraints and the ability to create custom 
+constraints. Bean Validation allows for validation of both JavaBean fields and 
+methods. For method-level validation, both the input parameters and return value 
+can be validated.
+
+Several new built-in constraints are included in Bean Validation 2.0, which 
```

Comment:
There're 3 occurrence of "new" in this paragraph. I think the 2nd and 3rd ones aren't needed.